python/docs/conf.py

```diff
@@ -334,3 +334,6 @@
 
 # If false, no index is generated.
 #epub_use_index = True
+
+# Skip sample endpoint link (not expected to resolve)
+linkcheck_ignore = [r'https://kinesis.us-east-1.amazonaws.com']
```

Comment:
I think it would be good to have for anyone that runs the link checking (since its in our docs Makefile) as it would be a junk error.